src/widgets/dialogs/EmotePopup.cpp

```diff
@@ -137,40 +161,62 @@ EmotePopup::EmotePopup(QWidget *parent)
     auto layout = new QVBoxLayout(this);
     this->getLayoutContainer()->setLayout(layout);
 
-    this->notebook_ = new Notebook(this);
-    layout->addWidget(this->notebook_);
-    layout->setMargin(0);
+    QRegExp searchRegex("\\S*");
+    searchRegex.setCaseSensitivity(Qt::CaseInsensitive);
+    QValidator *searchValidator = new QRegExpValidator(searchRegex);
```

Comment:
Should use [QRegularExpression](https://doc.qt.io/qt-5/qregularexpression.html) instead of `QRegExp`, and [QRegularExpressionValidator](https://doc.qt.io/qt-5/qregularexpressionvalidator.html) instead of `QRegExpValidator`

You'll also add an include to both at the top of the file